.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   test:
     docker:
       - image: continuumio/miniconda3
-      - image: postgres:11.5
+      - image: postgres:9
```

Comment:
What's the reason for downgrading postgres?

Reply:
It's because I'm using postgres9 for my own development environment and I want to make the environments consistent.
I don't see much difference between these two versions, but if you prefer a latest version, postgres11 should also be fine.